config/webpack/getThemeWebpackPromise.js

```diff
@@ -0,0 +1,120 @@
+const webpack = require('webpack');
+const MemoryFS = require('memory-fs');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const rtl = require('postcss-rtl');
+const Autoprefixer = require('autoprefixer');
+const PostCSSAssetsPlugin = require('postcss-assets-webpack-plugin');
+const PostCSSCustomProperties = require('postcss-custom-properties');
+const path = require('path');
+const Logger = require('../../scripts/utils/logger');
+
+/**
+ * This is a basic configuration to webpack just the theme file and grab the resulting css.  Thus,
+ * there are no JS, raw, etc. loaders. The resulting themeable variables and values will be populated in
+ * the cachedObject
+ * @param {String} rootPath the root path from where the webpack is being run
+ * @param {String} themeFile the file to be webpacked
+ * @param {Object} cachedObject this object will be populated with the resulting themeable variables and values
+ */
+const themeConfig = (rootPath, themeFile, cachedObject) => (
+  {
+    mode: 'production',
+    entry: {
```

Comment:
Do we need an entry point?

Reply:
Yes.  Webpack requires one as the entry point to the js then sass then css.